network/p2p/router_test.go

```diff
@@ -192,9 +191,9 @@ func TestAppRequestResponse(t *testing.T) {
 			sender := common.NewMockSender(ctrl)
 			handler := mocks.NewMockHandler(ctrl)
 			router := NewRouter(logging.NoLog{}, sender)
-			require.NoError(router.Connected(context.Background(), nodeID, nil))
-
-			client, err := router.RegisterAppProtocol(handlerID, handler)
+			sampler := &Peers{}
```

Comment:
this feels strange its clearly `peers` I get that it implements `Sample` is this `peerSampler`?

Reply:
I can just rename this to `peers` or `peerSampler`.